apps/design-system-docs/package.json

```
@@ -0,0 +1,52 @@
+{
+	"name": "design-system-docs",
```

I think our linting rules currently require this to be:

```suggestion
	"name": "@automattic/design-system-docs",
```

---

TIL, done in d477bb2dfa57ffde14555cb6ee33df978c237dfd